<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PJobFileEditor</class>
 <widget class="QMainWindow" name="PJobFileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PJOB Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PJobFileEditor.qrc">
    <normaloff>:/icons/Resources/icon128.png</normaloff>:/icons/Resources/icon128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentlyUsed">
     <property name="title">
      <string>Recently Used</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentlyUsed"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRuns">
    <property name="title">
     <string>Runs</string>
    </property>
    <addaction name="actionDelete_runs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRuns"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionDelete_runs">
   <property name="text">
    <string>Delete runs...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="PJobFileEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>PJobFileEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
